src/Accounts/Accounts/Az.Accounts.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.2.4'
```

Comment:
this version of module is not used, could you pls revert this change

Reply:
Hi, do you mean revert 2.2.8 to 2.2.4?